### DISCUSSION TITLE
In my GitHub images are not displaying

### DISCUSSION THREAD

**Discussion by kalyan-vurugonda** [community/community#33497] [Other Feature Feedback, Questions, & Ideas]
- In GitHub images are loading but not displaying on the page. From last week this was the problem with my GitHub account.
- Can anyone have a solution for this, please let me know

![Screenshot ](https://user-images.githubusercontent.com/69109920/191025088-794be3f7-29f3-4500-b5f7-e65b8f7a18c8.png)

**Comment by FahdSeddik** [comment]
Hey @kalyan-vurugonda 
This might be caused by several things. Make sure you can load the url directly from the src link in the readme file. If you still can't then it is most probably an issue with that link. If only you cannot view that link but other people can, then check your proxy or vpn. It might be blocking that src link. Hope this helps!
